be/src/olap/olap_cond.cpp

```diff
@@ -176,6 +150,14 @@ OLAPStatus Cond::init(const TCondition& tcond, const TabletColumn& column) {
                                  tcond.column_name.c_str(), operand.c_str(), op);
                 return res;
             }
+            if (min_value_field == nullptr || f->cmp(min_value_field) > 0) {
```

Comment:
I am a little confused.

`A->cmp(B) > 0` means `A > B`  or `A < B`?

Reply:
my bad, I will fix it  A->cmp(B) > 0 means A > B